config/database.yml

```diff
@@ -6,7 +6,11 @@ default: &default
 
 development:
   <<: *default
-  database: mastodon_development
+  database: <%= ENV['DB_DEV_NAME'] || 'mastodon_development' %>
+  username: <%= ENV['DB_USER'] || 'mastodon' %>
+  password: <%= ENV['DB_PASS'] || '' %>
+  host: <%= ENV['DB_HOST'] || 'localhost' %>
```

Comment:
In my case, Rails couldn't connect to PostgreSQL over a UNIX socket.  This changes it so the default is to connect over localhost; I'm not sure what the correct setting should be here to maintain the existing behaviour (while still allowing `DB_HOST` to work as an environment variable).